src/NServiceBus.Core/Performance/Statistics/ProcessingStatisticsBehavior.cs

```diff
@@ -20,13 +21,15 @@ class ProcessingStatisticsBehavior : IBehavior<IIncomingPhysicalMessageContext,
 
             state.ProcessingStarted = DateTime.UtcNow;
             context.Extensions.Set(state);
+            var stopwatch = Stopwatch.StartNew();
             try
             {
                 await next(context).ConfigureAwait(false);
             }
             finally
             {
-                state.ProcessingEnded = DateTime.UtcNow;
+                stopwatch.Stop();
+                state.ProcessingEnded = state.ProcessingStarted.AddTicks(stopwatch.ElapsedTicks);
```

Comment:
I don't think this works without some additional conversion. Stopwatch ticks aren't the same thing as DateTime ticks:

https://msdn.microsoft.com/en-us/library/system.diagnostics.stopwatch.elapsedticks(v=vs.110).aspx